packages/gatsby-source-shopify/.babelrc

```
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    ["../../.babel-preset.js"]
```

New change we recently added, but can we make this

```suggestion
    ["babel-preset-gatsby-package"]
```